pkgs/top-level/darwin-packages.nix

```diff
@@ -148,6 +148,8 @@ impure-cmds // appleSourcePackages // chooseLibs // {
 
   lsusb = callPackage ../os-specific/darwin/lsusb { };
 
+  MoltenVK = callPackage ../os-specific/darwin/MoltenVK { };
```

Comment:
```suggestion
  moltenvk = callPackage ../os-specific/darwin/moltenvk { };
```

Reply:
Wasn’t sure whether to follow the style of other frameworks or everything else. I’ve made the change. Thanks for the feedback!